packages/react/package.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "@parallelmarkets/react",
+  "version": "0.0.1",
+  "description": "ParallelMarkets.com React SDK",
+  "author": "Parallel Markets (https://parallelmarkets.com)",
+  "license": "MIT",
+  "homepage": "https://developer.parallelmarkets.com/docs/javascript",
+  "main": "dist/parallelmarkets-react.cjs.js",
+  "module": "dist/parallelmarkets-react.esm.js",
+  "scripts": {
+    "build": "NODE_ENV=production rollup -c",
+    "lint": "NODE_ENV=test eslint ./src && prettier -l ./src"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.15.0",
+    "@babel/eslint-parser": "^7.15.0",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-proposal-private-methods": "^7.14.5",
+    "@babel/plugin-proposal-private-property-in-object": "^7.14.5",
+    "@babel/plugin-syntax-jsx": "^7.14.5",
+    "@babel/preset-env": "^7.15.0",
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-commonjs": "^20.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.4",
+    "babel-preset-react-app": "^10.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.24.1",
+    "eslint-plugin-prettier": "^3.4.1",
+    "eslint-plugin-react": "^7.24.0",
+    "jest": "^27.0.6",
+    "prettier": "^2.3.2"
+  },
+  "peerDependencies": {
+    "@parallelmarkets/vanilla": "workspace:*",
```

I think this'll need to be the package version - otherwise I think `npm install ...` may fail
```suggestion
    "@parallelmarkets/vanilla": "^0.0.1",
```

---

When we publish, this will get replaced by the real version. I'll add the caret though! 290c45aaf179804a99c46e8ef984eb404d4be773